### PR TITLE
Tunning back-implementation for main entities

### PR DESCRIPTION
**Pull request by AndrePu**

### CODE REVIEW

File: Cooper/DAO/ChatDAO.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            ChatDb chat = Get(id);
+
+            //chat.PlayersList = GetPlayersList(id);
+
+            return chat;
+        }
+
+        public IEnumerable<ChatDb> GetAll()
+        {
+            List<ChatDb> userReviews = new List<ChatDb>();
```
Comment (Hunroll) [inline Cooper/DAO/ChatDAO.cs]: Not renamed)

File: Cooper/DAO/Mapping/EntityMapping.cs [hunk 2]
```diff
@@ -212,7 +286,192 @@ public static EntityORM Map(UserDb user, HashSet<string> attributes)
...
+                    break;
+                case "11":
+                    month = "NOV";
+                    break;
+                case "12":
+                    month = "DEC";
+                    break;
+            }
+
+            string oracleDateFormat = $"{day}-{month}-{year}";
```
Comment (Hunroll) [inline Cooper/DAO/Mapping/EntityMapping.cs]: dt.ToString("dd-MMM-yyyy")

File: Cooper/DAO/ChatDAO.cs (hunk 1 above)
Comment (AndrePu) [reply Cooper/DAO/ChatDAO.cs]: Fixed for Chat, Message DAO

File: Cooper/DAO/Mapping/EntityMapping.cs (hunk 2 above)
Comment (AndrePu) [reply Cooper/DAO/Mapping/EntityMapping.cs]: Removed corresponding method for converting date

**Pull request merged**